### PR TITLE
[feature/MORTAR-275-4] bug fix - duration, remove height/width

### PR DESCRIPTION
**Pull request by bpearsoncse**
https://jira.natgeo.com/browse/MORTAR-275

Small change to fix an issue with unit test that was breaking build to QA

### CODE REVIEW

**Comment by bpearsoncse** [comment]
Changes made - passing back to you @asabaylus

File: app/modules/promocard/components/shared/PromoImage.jsx [hunk 1]
```diff
@@ -35,8 +35,6 @@ PromoImage.PropTypes = {
   leadMedia: PropTypes.shape({
     url: PropTypes.string,
     aspectRatio: PropTypes.number,
-    height: PropTypes.number,
```
Comment (asabaylus) [inline app/modules/promocard/components/shared/PromoImage.jsx]: Looking good glad to reduce complexity based on my talking with Ben we should not need height / width for the content packages

**Comment by asabaylus** [comment]
@bpearsoncse I dont see anything related to the CTA button. Is that deliberate?

**Comment by asabaylus** [comment]
Also modal isnt doing anything

**Comment by bpearsoncse** [comment]
@asabaylus - CTA and modal is deliberate for this ticket (I had commented earlier today in the ticket). Updating PromoImage.jsx now

**Comment by bpearsoncse** [comment]
@asabaylus - updated to add height/width back in to PromoImage.jsx and Article.jsx

**Comment by asabaylus** [comment]
Shipping!

**Pull request merged**